Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.jquery.ts

```diff
@@ -1486,11 +1537,15 @@
         var documentWidth = number($(window).width());
         var documentHeight = number($(window).height());
 
-        var space = new Object();
-        space.top = offset.top - scrollTop;
-        space.bottom = documentHeight - ((offset.top + height) - scrollTop);
-        space.left = offset.left - scrollLeft;
-        space.right = documentWidth - ((offset.left + width) - scrollLeft);
+        var space: Space = {
```

Comment:
That type is not quite correct because the `spaceVertical`, `spaceHorizontal`, and `space` are actually never null. For now just do an `any` type or save all the properties in variables and create the type from that. This way we can remove the nullability from the `Space` type.